### ISSUE TITLE
Abandon CircleCI in favor of github actions 

### ISSUE THREAD

**Issue by FedeDP** [falcosecurity/libs#1384]
**Motivation**

All the falcosecurity org moved to github actions, retiring CircleCI. We should move away libs from CircleCI too, for consistency.
For now, circleCI is just used to test arm64 support since it provides arm64 machines.
Therefore we first need an arm64 runner; either we wait for github actions to finally provide linux arm runners or we use our own test-infra nodes (like we do for Falco).

This is just to track the issue.

**Comment by FedeDP** [comment]
See https://github.com/actions/runner-images/issues/5631

**Comment by Andreagit97** [comment]
yeah absolutely we cannot remove circle CI until we have an arm node :/

**Comment by LucaGuerra** [comment]
Since we started using the CNCF Community bare metal cluster for kernel testing, it could be a good resource that we could consider. While we already have approval for the kernel testing project that we're running, I wanted to make sure we could start another node if necessary and if we need to ask permission beforehand. This is the issue on the topic: https://github.com/cncf/cluster/issues/251

**Comment by vielmetti** [comment]
I'll note that it's relatively expensive for the CNCF CIL to have an entire 80-core bare metal node used 24/7 just for occasional sporadic Github Actions runs.

**Comment by FedeDP** [comment]
> I'll note that it's relatively expensive for the CNCF CIL to have an entire 80-core bare metal node used 24/7 just for occasional sporadic Github Actions runs.

I agree! The [kernel-testing](https://github.com/falcosecurity/kernel-testing) framework  will, sooner or later, be enabled on PRs, master branch and releases, concurrently.
It's just that the tool we use today to manage firecracker vms (ie: https://github.com/weaveworks/ignite) does not support concurrent runs.

**Comment by FedeDP** [comment]
cc @Andreagit97 
latest kernel-testing updates brought a new ubuntu test image that uses the exactly same image used by our remaining circleCI job: https://github.com/falcosecurity/kernel-testing/commit/3f359211f1155a3cf91046bef7befc914505c4c6
Even if we only run `scap-open` and not `drivers_test` for the moment, i think we can safely kill the last job and drop circleCI (after all we are concerned about verifier issues).
WDYT?

**Comment by Andreagit97** [comment]
yes makes sense! i think we can drop circleCI
